other/edge-server/README.md

```diff
@@ -1,72 +1,86 @@
-  get Edge Server enabled for your new app.
+1. Create an App Services App based on the `edge-server.todo` template app, and
+   get Edge Server enabled for your new app.
 2. Download, configure, and start the Edge Server.
 3. Install dependencies and start the Node.js Express server.
 4. Install dependencies and start the React server.
 
-This is a MERN stack that connects to the Edge Server instead of Atlas. 
-In this version of the example app, MongoDB wire protocol connections connect 
-directly to Edge Server without authorization. 
+This is a MERN stack that connects to the Edge Server instead of Atlas.
+In this version of the example app, MongoDB wire protocol connections connect
+directly to Edge Server without authorization.
 
-For a MERN stack tutorial, refer to 
+For a MERN stack tutorial, refer to
 [How to Use Mern Stack: A Complete Guide](https://www.mongodb.com/languages/mern-stack-tutorial).
 
-For more information about Edge Server, refer to the 
+For more information about Edge Server, refer to the
 [Edge Server documentation](https://www.mongodb.com/docs/atlas/app-services/edge-server/).
 
 Atlas Edge Server is currently in Private Preview. To learn more about
 previewing Edge Server, refer to the product page at
 [Edge Server](https://www.mongodb.com/products/platform/atlas-edge-server).
 
+## Quick Start
+
+1. Create an App Services App based on the `edge-server.todo` template app, and
+   get Edge Server enabled for your new app.
+2. Download and configure the Edge Server.
+3. Start Docker
+4. From the project's root directory, run `npm run install`.
+5. Then run `npm run start`.
+6. Explore how CRUD works from both the React Client and the App Services UI.
+
+When you're done with the template app, make sure to shut everything down,
+including the Edge Server. From the project's root directory, run
+`npm run shut-down`.
+
 ## Create an App Services App
 
-Edge Server connects to Atlas through an App Services App backend. 
-For this example application, you'll create an App Services App based on the 
-`edge-server.todo` template app. 
+Edge Server connects to Atlas through an App Services App backend.
+For this example application, you'll create an App Services App based on the
+`edge-server.todo` template app.
 
-This template app uses a backend with Atlas Device Sync enabled and configured with: 
+This template app uses a backend with Atlas Device Sync enabled and configured with:
 
 - An already defined `Item` collection in a `todo` database
-- A single default rule that allows *any* user to read or write Items to 
+- A single default rule that allows _any_ user to read or write Items to
   the collection
 
-> **NOTE:** This example application is NOT interoperable with any of the Atlas 
-> Device SDK template apps. You must use the `edge-server.todo` template app. 
+> **NOTE:** This example application is NOT interoperable with any of the Atlas
```

Comment:
I _think_ we can take this note out now. By adding the `owner_id` back, we should have made it interoperable with the Device SDK template apps. (We should also test this backend with the Device SDK template apps.)

You must still use `edge-server.todo` to _create_ a template app that the Edge Server template can connect to, but this should now work with the Device SDK templates.